tokio/src/io/mod.rs

```diff
@@ -217,6 +217,13 @@ cfg_io_driver_impl! {
     pub(crate) use poll_evented::PollEvented;
 }
 
+cfg_aio! {
+    /// BSD-specific I/O types
+    pub mod bsd {
+        pub mod poll_aio;
+    }
+}
```

Comment:
For symmetry with `AsyncFd`, it seems like we should put the types directly in the `bsd` module.

Reply:
moved.